packages/__docs__/src/Properties/index.js

```diff
@@ -206,8 +206,8 @@ class Properties extends Component {
       return <span>{type.value}</span>
     }
 
-    const values = type.value.map(({ value }) => (
-      <li css={styles.listItem} key={value}>
+    const values = type.value.map(({ value }, idx) => (
+      <li css={styles.listItem} key={idx}>
```

Comment:
These needed to be fixed, because the values could clash as identical keys in the .map()